cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/PushUtils.java

```diff
@@ -494,11 +497,14 @@ private static String getAccessTokenOfCLI() {
      */
     public static void pushAllModules(Path sourceRootPath) {
         try {
-            List<String> fileList = Files.list(sourceRootPath.resolve(ProjectDirConstants.SOURCE_DIR_NAME))
-                                         .filter(path -> Files.isDirectory(path, LinkOption.NOFOLLOW_LINKS))
-                                         .map(ProjectDirs::getLastComp)
-                                         .filter(dirName -> !isSpecialDirectory(dirName))
-                                         .map(Path::toString).collect(Collectors.toList());
+            List<String> fileList;
+            try (Stream<Path> stream = Files.list(sourceRootPath.resolve(ProjectDirConstants.SOURCE_DIR_NAME))) {
```

Comment:
Instead of "stream" shall we use "moduleDirsStream". "stream" is too vague for me.

Reply:
Renamed variable.